Makefile

```diff
@@ -50,7 +50,7 @@ clean:
 
 .PHONY: gen
 gen:
-	go generate ./core/...
+	go generate
```

Comment:
I've tried `go generate ./... ` however this caused issues with vendor code - you could for now just explicitly generate the coredns.go